packages/ffe-form-react/src/index.d.ts

```diff
@@ -18,6 +18,7 @@ export interface BaseFieldMessageProps extends React.ComponentProps<'div'> {
     children: React.ReactNode;
     className?: string;
     element?: string;
+    role?: string;
```

Comment:
Skjønner ikke helt forskjellen på denne og propTypes definert i BaseFieldMessage, men bør man gjøre samme endring her? Der det er "one of" istedenfor bare string?

Reply:
Forskjellen er i grunn bare at dette er typescript. Fikset proptypen her også.